src/v/config/configuration.cc

```diff
@@ -1190,7 +1299,7 @@ configuration::configuration()
       *this,
       "storage_max_concurrent_replay",
       "Maximum number of partitions' logs that will be replayed concurrently "
-      "at startup, or flushed concurrently on shutdown.",
+      "at startup, or flushed concurrently on shutdown. asdasdasdasdas",
```

Comment:
:eyes:

Reply:
@Deflaimun

Reply:
Oops, not sure how that got in there. Will remove! Thanks for the catch

Reply:
Fixed!